.circleci/real_config.yml

```diff
@@ -70,24 +68,31 @@ release-and-rc-filters: &release-and-rc-filters
       - /.*/
   tags:
     only:
-      - /(\d)+(\.(\d)+)+/
-      - /((\d)+(\.(\d)+)+)(-rc)(\d)+/
+      - /v\d+\.\d+\.\d+(rc\d+)?/
```

Comment:
this would be a great place for a yaml anchor considering you could give this pattern a name

Reply:
Noted, although I'm hesitant to use an alias within an anchor definition, especially when these particular regular expressions only appear once. It also makes it easier to see what's being defined, instead of having to follow another link back to the regular expression itself.

Reply:
nested anchors work fine syntactically. i just think it's nice to see a label next to a regex, even if it's a comment.

Reply:
I've added some comments to each tag-specific regex. Let me know what you think.

Reply:
lgtm